masterlist.yaml

```
@@ -1709,6 +1709,17 @@ plugins:
     group: *fixesGroup
     msg: [ *alreadyInUFO4P ]
 
+  - name: 'ModdableGoodfeels.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/11500' ]
```

Trailing slash for consistency:

```suggestion
    url: [ 'https://www.nexusmods.com/fallout4/mods/11500/' ]
```